modules/end-user-guide/examples/che-ref_a-kubernetes-secret-as-an-variable.adoc

```diff
@@ -16,28 +10,19 @@ metadata:
     app.kubernetes.io/component: workspace-secret
 ...
 ----
-+
-Annotations must indicate that the given secret is mounted as an environment variable, provides variable names, and optionally, specifies the container name where this mount will be applied. If there is no target-container annotation defined, the secret will be mounted into all user containers of the {prod-short} workspace, but this is applicable *_only for the {prod-short} version 7.14_*.
-+
-[source,yaml]
-----
-apiVersion: v1
-kind: Secret
-metadata:
-  name: mvn-settings-secret
-  annotations:
-    che.eclipse.org/target-container: maven
-    che.eclipse.org/env-name: FOO_ENV
-    che.eclipse.org/mount-as: env
-  labels:
-   ...
-data:
-  mykey: myvalue
-----
-+
-This results in the environment variable named `FOO_ENV` and the value `myvalue` being provisioned into the `maven` container.
-+
-*_Since the {prod-short} version 7.15_*, the `target-container` annotation is deprecated and `automount-workspace-secret` annotation with boolean values is introduced. Its purpose is to define the default secret mounting behavior, with the ability to be overridden in a devfile. The `true` value enables automatic mounting into all workspace containers. In contrast, the `false` value disables the mounting process until it is explicitly requested in a devfile component using the `automountWorkspaceSecrets:true` property.
+
+// The corresponding procedure is underdeveloped. Parts of this example will likely end up in the procedure. A comprehensive rewrite is needed. max-cx
+
+* The following example shows you how to mount the secret as an environment variable by including the `<url>/mount-as: env` annotation.
+
+* The secret's environment variable (`/env-name:`) is declared (`FOO_ENV`) in the `annotations`.
+* The `/automount-workspace-secret` annotation with a boolean value defines the default secret mounting behavior (that can be overridden in a devfile):
+
+** `true` enables automatic mounting into all workspace containers.
+
+** `false` disables the mounting process until it is explicitly requested in a devfile component using the `automountWorkspaceSecrets:true` property.
+
+* In the next example, the environment variable `FOO_ENV` and the value `myvalue` will be provisioned into all workspace containers.
```

Comment:
```suggestion
* The following example provisions the `FOO_ENV` environment variable and `myvalue` value into all workspace containers.
```

Reply:
Avoid future tense and passive voice.

Reply:
Use commas to indicate parenthetical remarks: `the environment variable, `FOO_ENV`, and...` 
Or, avoid parenthetical remarks altogether: `the `FOO_ENV` environment variable and..."

Reply:
Yes, "the 'FOO_ENV' environment variable" is the best option here. Thank you.